src/MongoDB/Exception/CommandException.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2018 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <php.h>
+
+#include "phongo_compat.h"
+#include "php_phongo.h"
+
+zend_class_entry* php_phongo_commandexception_ce;
+
+/* {{{ proto document MongoDB\Driver\Exception\CommandException::getResultDocument()
+   Returns the result document from the failed command. */
+static PHP_METHOD(CommandException, getResultDocument)
```

Comment:
In the interest of improving test coverage, I think it'd be a good idea to create `tests/exception/commandexception-getresultdocument-001.phpt` for this.

The test won't need to use servers, so you can do without a `--SKIPIF--` and the typical "basic" include. I'd suggest creating a new instance of the exception (message/code is irrelevant, so no constructor arguments), using [reflection](http://php.net/reflection) to assign an arbitrary document to the `resultDocument` property, and then verifying that this method returns the same document. For the comparison, I'd suggest starting with `var_dump($resultDocument === $exception->getResultDocument())` and asserting that it outputs "bool(true)". Based on the implementation, I think an assigned object (assuming stdClass is used as the document) would be the same. `===` should certainly work with an array document.

If this works out, you can then copy the test for WriteException (either extend it locally in the test or use BulkWriteException), too.